src/LSST/faultmgr/FaultStatusBits.cpp

```diff
@@ -261,6 +270,114 @@ uint64_t FaultStatusBits::getMaskPowerSubsystemFaults(control::PowerSystemType s
             + getPowerSystemTypeStr(sysType) + " unexpected type");
 }
 
+
+uint64_t FaultStatusBits::getTelemetryFaultManagerAffectedFaultMask() {
+    if (_telemetryFaultManagerAffectedFaultMask == nullptr) {
+        // Starts with nothing set
+        FaultStatusBits::Ptr fsm(new FaultStatusBits());
+        lock_guard<mutex> lg(faultMaskCreationMtx);
+        if (_telemetryFaultManagerAffectedFaultMask == nullptr) {
+            // - "Affected Fault Mask":
+            fsm->setBitAt(ACTUATOR_LIMIT_CL);          // “Actuator limit CL”
+            fsm->setBitAt(INCLINOMETER_W_LUT);         // “Inclinometer error w/ lut”
+            fsm->setBitAt(INCLINOMETER_RANGE_ERR);     // “Inclinometer range error”
+            fsm->setBitAt(MIRROR_TEMP_SENSOR_FAULT);   // “mirror temp sensor fault”
+            fsm->setBitAt(ELEVATION_ANGLE_DIFF_FAULT); // “elevation angle difference error fault”
+            fsm->setBitAt(ACTUATOR_FAULT);             // “actuator fault”
+            fsm->setBitAt(EXCESSIVE_FORCE);            // “excessive force”
+            fsm->setBitAt(AXIAL_ACTUATOR_ENCODER_RANGE_FAULT);   // “axial actuator encoder range fault”
+            fsm->setBitAt(TANGENT_ACTUATOR_ENCODER_RANGE_FAULT); // “tangent actuator encoder range fault”
+            fsm->setBitAt(TANGENT_LOAD_CELL_FAULT);    // “tangent load cell fault”
+            _telemetryFaultManagerAffectedFaultMask = fsm;
+        }
+    }
+    return  _telemetryFaultManagerAffectedFaultMask->_bitmap;
+}
+
+uint64_t FaultStatusBits::getTelemetryFaultManagerAffectedWarningMask() {
+    if (_telemetryFaultManagerAffectedWarningMask == nullptr) {
+        // Starts with nothing set
+        FaultStatusBits::Ptr fsm(new FaultStatusBits());
+        lock_guard<mutex> lg(faultMaskCreationMtx);
+        if (_telemetryFaultManagerAffectedWarningMask == nullptr) {
+            // - "Affected Warning Mask":
+            // warnings
+            fsm->setBitAt(MONITOR_ILC_READ_WARN);         // “monitoring ILC read error warning”
+            fsm->setBitAt(ACTUATOR_LIMIT_OL);             // “actuator limit OL”
```

Comment:
OL means opened-loop

Reply:
"actuator limit OL" can be searched for in LabView. I added  opened-loop to the comment.